src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -573,17 +573,9 @@ private void buildUnregisteredAndUnrepresentedDefendant(EventHistory.EventHistor
 
     private void buildAcknowledgementOfServiceReceived(EventHistory.EventHistoryBuilder builder, CaseData caseData) {
         LocalDateTime dateAcknowledge = caseData.getRespondent1AcknowledgeNotificationDate();
-        String responseIntentionType = caseData.getRespondent1ClaimResponseIntentionType().getLabel();
-
-        if ((dateAcknowledge == null) && (caseData.getRespondent2AcknowledgeNotificationDate() == null)) {
+        if (dateAcknowledge == null) {
             return;
         }
-
-        if (caseData.getRespondent2AcknowledgeNotificationDate() != null) {
-            dateAcknowledge = caseData.getRespondent2AcknowledgeNotificationDate();
-            responseIntentionType = caseData.getRespondent2ClaimResponseIntentionType().getLabel();
-        }
-
         builder
```

Comment:
Can you please add a TODO comment here saying that Defendant 2 will be handled under RPA ticket [CMC-1677]

Reply:
added comment